.github/dependabot.yml

```diff
@@ -12,3 +12,57 @@ updates:
     interval: daily
   open-pull-requests-limit: 3
   rebase-strategy: disabled
+- package-ecosystem: gomod
+  directory: "/examples/cockroachdb"
```

Comment:
remove quotes :)

```suggestion
  directory: /examples/cockroachdb
```

Reply:
Removed in [a401b2d](https://github.com/testcontainers/testcontainers-go/pull/654/commits/a401b2d8f471173ec9b6a932ba56f5aeab39f332)